ruby/ext/google/protobuf_c/message.c

```diff
@@ -1171,6 +1171,11 @@ static VALUE Message_encode_json(int argc, VALUE* argv, VALUE klass) {
                               Qfalse))) {
       options |= upb_JsonEncode_EmitDefaults;
     }
+
+    if (RTEST(rb_hash_lookup2(hash_args, ID2SYM(rb_intern("enums_as_integers")),
```

Comment:
This name should be `format_enums_as_integers`, per our internal document on this new API.

Reply:
thank you for the suggestion, updated.